modules/optimeraBidAdapter.js

```diff
@@ -0,0 +1,82 @@
+import {registerBidder} from 'src/adapters/bidderFactory';
+const BIDDER_CODE = 'optimera';
+const SCORES_BASE_URL = 'https://s3.amazonaws.com/elasticbeanstalk-us-east-1-397719490216/json/client/';
+
+export const spec = {
+  code: BIDDER_CODE,
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {bidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function (bidRequest) {
+    if (typeof bidRequest.params.custom != 'undefined' && typeof bidRequest.params.custom.clientID != 'undefined') {
+      return true;
+    } else {
+      return false;
+    }
+  },
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {validBidRequests[]} - an array of bids
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function (validBidRequests) {
+    var optimeraHost = window.location.host;
+    var optimeraPathName = window.location.pathname;
+    var timestamp = Math.round(new Date().getTime() / 1000);
+    if (typeof validBidRequests[0].params.custom.clientID != 'undefined') {
+      var clientID = validBidRequests[0].params.custom.clientID;
+      var scoresURL = SCORES_BASE_URL + clientID + '/' + optimeraHost + optimeraPathName + '.js';
+      return {
+        method: 'GET',
+        url: scoresURL,
+        payload: validBidRequests,
+        data: {'t': timestamp}
+      };
+    }
+  },
+  /**
+   * Unpack the response from the server into a list of bids.
+   *
+   * Some required bid params are not needed for this so default
+   * values are used.
+   *
+   * @param {*} serverResponse A successful response from the server.
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function (serverResponse, bidRequest) {
+    var scores = serverResponse.body.replace('window.oVa = ', '');
+    scores = scores.replace(';', '');
+    scores = JSON.parse(scores);
+    var validBids = bidRequest.payload;
+    var bidResponses = [];
+    var dealId = '';
+    for (var i = 0; i < validBids.length; i++) {
+      if (typeof validBids[i].params.custom.clientID != 'undefined') {
+        if (validBids[i].adUnitCode in scores) {
+          dealId = scores[validBids[i].adUnitCode];
+        }
+        var bidResponse = {
+          bidderCode: spec.code,
```

Comment:
`bidderCode` will be set automatically by `bidderFactory` now, this line can be dropped

Reply:
@matthewlane ok, I removed the bidderCode. Yes, that endpoint is correct, we just had an error and that has been restored. The Travis build failed, but passes on my local. Can you rerun the build? Thank you.